src/Query/SqlWalker.php

```diff
@@ -1576,18 +1585,30 @@ public function walkNewObject(AST\NewObjectExpression $newObjectExpression, stri
                     $sqlSelectExpressions[] = trim($e->dispatch($this));
                     break;
 
+                case $e instanceof AST\AllFieldsExpression:
+                    if (! $newObjectExpression->hasNamedArgs) {
+                        throw new LogicException('All fields expressions must be used with named arguments dto constructor.');
```

Comment:
```suggestion
                        throw new LogicException('All fields Expression must be used with named arguments DTO constructor.');
```

For consistency with the other `LogicException`.